src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp

```diff
@@ -472,15 +468,15 @@ class CoupledSolver : public SolverBase
 
         if( isConverged )
         {
+          // we still want to count current iteration
```

Comment:
Not sure we need this. We might exit the loop with `iter==maxIterNewton` which may be reported somewhere else.

Reply:
the time step cut logic below relies on `isConverged` flag
i think it makes sense to count the iteration that was performed, and in current implementation it will only affect the time step selector (currently selector sees 0 as number of iterations when outer loop converges in one iteration)